docs/COREDUMP_PACKAGE_METADATA.md

```diff
@@ -1,149 +1,4 @@
 ---
-title: Package Metadata for Core Files
-category: Interfaces
-layout: default
-SPDX-License-Identifier: LGPL-2.1-or-later
+layout: forward
+target: ELF_PACKAGE_METADATA
```

Comment:
target needs to be absolute, `/ELF_PACKAGE_METADATA`, to avoid a calculated destination like:  `/COREDUMP_PACKAGE_METADATA/ELF_PACKAGE_METADATA`